scripts/extract-resume-linkedin.py

```
@@ -0,0 +1,193 @@
+
+import argparse
+import json
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from resume_utils import fmt_date
+
+
+def fmt_date_range(start: str, end: str | None) -> str:
+    """Format a start–end date range for display."""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)